deploifai/utilities/server_state.py

```diff
@@ -0,0 +1,81 @@
+import click
+from PyInquirer import prompt
+
+from deploifai.api import DeploifaiAPIError
+from deploifai.context import pass_deploifai_context_obj, DeploifaiContextObj
+
+
+@pass_deploifai_context_obj
+def change_state(context: DeploifaiContextObj, command: str, ):
+
+    current_workspace = context.global_config["WORKSPACE"]["username"]
+
+    click.secho("Workspace Name: {}".format(current_workspace), fg="blue")
+
+    project_id = context.local_config["PROJECT"]["id"]
+    where_project = {"id": {"equals": project_id}}
+
+    fragment = """
+                fragment project on Project {
+                trainings{
+                        id name status state
+                    }
+                }
+                """
+    projects_data = context.api.get_projects(workspace=current_workspace, where_project=where_project, fragment=fragment)
```

Comment:
Should use a specific `get_project` api call as well

Reply:
Yes
Refactored

Reply:
Using the `get_training_server` api call instead

Reply:
Yes
Refactored